src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -744,9 +744,13 @@ internal bool TryResolveFilePathFromSolution(string solutionPath, string pathFro
                     return true;
                 }
             }
-            catch
+            catch (Exception ex) when (ex is ArgumentException ||
+                                       ex is IOException ||
+                                       ex is PathTooLongException ||
+                                       ex is UnauthorizedAccessException)
             {
                 // do not threw exception so that it keeps trying to resolve path the next
+                Trace.WriteLine($"{nameof(TryResolveFilePathFromSolution)} threw exception: {ex.Message}");
```

Comment:
>Message [](http://example.com/codeflow?start=96&length=7)

replace 'ex.Message' with simply 'ex' (which is a synonym for ex.ToString(). If you do this, the generated string will contain not only the exception message but also the stack, which is usually essential data for debugging.

Reply:
was thinking not print too many details to the user. we can add an option or --verbose switch to control the log behavior.